### DISCUSSION TITLE
Is there a way to set a style formatter for CSS/SCSS files?

### DISCUSSION THREAD

**Discussion by denkhis** [zed-industries/zed#10521] [Feature Requests]
I edited my `settings.json` config like that, but it had no effect neither for `*.css` nor for `*.scss` files.
The `file_types` setting requires for `.scss` syntax highlighting, which is doesn't work by default.

```json
  "languages": {
    "CSS": {
      "code_actions_on_format": {
        "source.fixAll.stylelint": true
      }
    },
    "SCSS": {
      "code_actions_on_format": {
        "source.fixAll.stylelint": true
      }
    }
  },
  "file_types": {
    "CSS": ["scss"]
  }
```

**Comment by Igoryas** [comment]
```json
{
  "languages": {
    "CSS": {
      "formatter": {
        "code_actions": {
          "source.fixAll.stylelint": true
        }
      }
    },
    "SCSS": {
      "formatter": {
        "code_actions": {
          "source.fixAll.stylelint": true
        }
      }
    }
  },
  "file_types": {
    "CSS": ["scss"]
  }
}
```

**Reply by globalexport** [reply]
Hi Igoryas,
 
Thanks for sharing your config.

I installed stylelint in my project:
```
yarn add -D stylelint stylelint-config-standard-scss
```
Afterwards, I created a new `stylelintrc.json` file in my project's root:
```
{
  "extends": "stylelint-config-standard-scss"
}
``` 

Then I added your config to my zed config. Sadly, I see no difference. 

Do you have an idea what I may be missing here?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
